src/game_api/script/lua_vm.cpp

```diff
@@ -412,7 +412,6 @@ end
             if (!import_backend->get_enabled())
             {
                 import_backend->set_enabled(true);
-                import_backend->update();
```

Comment:
Why was this previously updating? Presumably it did that to run any code at global scope. So is it really safe to stop doing this?

Reply:
"run any code at global scope" - i don't quite get that

This will run the update out of sync, but i revert that for now